packages/smart-router/evm/v3-router/types/trade.ts

```diff
@@ -13,7 +13,7 @@ export interface SmartRouterTrade<TTradeType extends TradeType> {
   // From routes we know how many splits and what percentage does each split take
   routes: Route[]
 
-  gasEstimate: bigint
+  gasEstimate?: bigint
```

Comment:
The gas estimate should be a required field. Can we check from which place it's optional?